src/config.rs

```diff
@@ -558,26 +554,27 @@ impl Config {
       }
     };
 
-    let shell_args = if matches.is_present(arg::CLEAR_SHELL_ARGS) {
-      Vec::new()
-    } else {
+    let shell_args = if matches.is_present(arg::SHELL_ARG) {
       matches
         .values_of(arg::SHELL_ARG)
         .unwrap()
```

Comment:
Instead of testing for existence and then unwrapping, this can be replaced with an `if let`:

```suggestion
    let shell_args = if let Some(shell_args) = matches.values_of(arg::SHELL_ARG) {
        shell_args
```

Reply:
Actually clippy want's us to use
```rust
    let shell_args = matches
      .values_of(arg::SHELL_ARG)
      .map_or(Vec::new(), |shell_args| {
        shell_args.map(str::to_owned).collect()
      });
```